hw/xilinx/ips/hpc/xlnx_axi_dwidth_32to512_converter/config.tcl

```diff
@@ -0,0 +1,10 @@
+# Author: Manuel Maddaluno <manuel.maddaluno@unina.it>
+# Description: AXI Data Width Converter - Needed to link the AXI crossbar (in general the entire SoC 32 bits) to the DDR MIG (512 bit)
+
+create_ip -name axi_dwidth_converter -vendor xilinx.com -library ip -version 2.1 -module_name $::env(IP_NAME)
+
+set_property -dict [list \
+  CONFIG.MI_DATA_WIDTH {512} \
+  CONFIG.SI_DATA_WIDTH {32} \
```

Comment:
for SI_ID_WIDTH and SI_DATA_WIDTH , use config-based envvar, like in https://github.com/MaistoV/UninaSoC/blob/main/hw/xilinx/ips/embedded/xlnx_jtag_axi/config.tcl#L14

Reply:
see https://github.com/MaistoV/UninaSoC/pull/25#discussion_r1886852479